libevmasm/LinkerObject.h

```diff
@@ -58,6 +58,16 @@ struct LinkerObject
 	/// Bytecode offsets of named tags like function entry points.
 	std::map<std::string, FunctionDebugData> functionDebugData;
 
+	struct SubAssembly {
+		size_t start;
+		size_t length;
+		bool isCreation;
+		std::string bytecode;
```

Comment:
It's not the same thing as `bytecode` stored directly in `LinkerObject`. The name should contain enough information to make that clear:

```suggestion
		std::string linkedBytecodeHex;
```

Reply:
Actually, why do you even store it? It's relatively large (on the same order as the size of the whole `LinkerObject`) and all you ever do with it is check its size (which you could get from `length` instead).

Reply:
I used it for debugging - the actual bytecode string is unnecessary as far as the feature is concerned. I'll add a task in the PR as a reminder to remove it.